code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -578,10 +578,10 @@ func (c *Chain) setupInitialState(initStates *state.InitStates, creationDate com
 	stateCtx := cstate.NewStateContext(nil, pmt, nil, nil, nil, nil, nil, nil, nil)
 	mustInitPartitions(stateCtx)
 
-	if err := c.addInitialStakes(initStates.Stakes, creationDate, stateCtx); err != nil {
-		logging.Logger.Error("init stake failed", zap.Error(err))
-		panic(err)
-	}
+	// if err := c.addInitialStakes(initStates.Stakes, creationDate, stateCtx); err != nil {
+	// 	logging.Logger.Error("init stake failed", zap.Error(err))
+	// 	panic(err)
+	// }
```

Comment:
Should not comment out this, otherwise the state will be empty

Reply:
This code needs to be reviewed. We're unable to start a network with no blocks because sharders are unable to operate. Could you look into it? Or just accept this change and postpone the fix.

Reply:
I can see this PR https://github.com/0chain/0chain/pull/1911 has a fix. Uncommenting.

Reply:
should be fixed in staging